locales/en-US.json

```diff
@@ -484,5 +484,6 @@
     "channel_tab_streams_label": "Livestreams",
     "channel_tab_playlists_label": "Playlists",
     "channel_tab_community_label": "Community",
-    "channel_tab_channels_label": "Channels"
+    "channel_tab_channels_label": "Channels",
+    "temporary_video_not_available_explanation": "Can't load the video. YouTube is currently trying to block Invidious instances. <a href=\"`x`\">Click here for more info about the issue.</a>"
```

Comment:
I wouldn't add a translation string, as it will be picked up by weblate, and it'll take a while to translate in all langues, then to remove when this is (hopefully) over.